### DISCUSSION TITLE
v2023.09.06

### DISCUSSION THREAD

**Discussion by aperezbios** [ZuluSCSI/ZuluSCSI-firmware#292] [Announcements]
## What's Changed
* README: use truncate instead of fallocate in Linux example by @PetteriAimonen in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/277
* RP2040 Initiator mode: set initiator ID in selection phase (#262, #264) by @PetteriAimonen in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/280
* Post UNIT_ATTENTION condition after CD medium change (fixes #247) by @PetteriAimonen in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/283
* Rename files for new pcb designs using the RP2040 by @morio in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/284
* New Xebec S1410 vendor command and various work by @jjakob in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/286
* Log Xebec vendor command and decoded data by @jjakob in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/287
* Set vendor / product id with image file name by @phe78 in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/290
* Fixes for Apple vendor modepage sending by @PetteriAimonen in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/278

## Previous changes in v2023.07.01

* Improve CD-ROM image switching logic by @PetteriAimonen in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/251
* Increase MPC3000 InitPreDelay to 600ms, from 500ms (https://github.com/ZuluSCSI/ZuluSCSI-firmware/commit/cb5fd0a72c12b561aa523e40011334596bfefdcb)
* Fix bootloader size comparison (fix #227) by @PetteriAimonen in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/250
* RP2040 SDIO: Increase stopTransmission timeout. by @PetteriAimonen in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/253
* ZuluSCSI GD32 V1.1: Fix PhyMode=2 (DMA_TIMER) with AHA2940UW by @PetteriAimonen in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/254
* ZuluSCSI GD32 V1.1 Fix PhyMode=1(PIO) (fix #245) by @PetteriAimonen in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/249
* Temporary fix for DMA_TIMER mode on ZuluSCSI v1.1 by @morio in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/256

### The RP2040 Audio firmware/features can only be used with additional hardware, which is not yet commercially available.

* EXPERIMENTAL SP/DIF output on expansion pins of RP2040 boards by @saybur in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/201
* EXPERIMENTAL audio playback support from BIN/CUE files. by @saybur in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/207

For boards marked:

* "ZuluSCSI V1.1", you must download and place ZuluSCSIv1_1_.bin on your SD card.
* "ZuluSCSI V1.0", you must download and place ZuluSCSIv1_0_.bin on your SD card.
* "ZuluSCSI Mini V1.0" (DB25 version, blue printed circuit board) ZuluSCSIv1_0_mini.bin 
* "ZuluSCSI RP2040" (ALL red boards), you must download and place ZuluSCSI_RP2040.bin on your SD card.
* "ZuluSCSI Compact Homebrew" (Green or Red boards), you must download and place ZuluSCSI_RP2040.bin on your SD card.

## New Contributors
* @jjakob made their first contribution in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/286
* @phe78 made their first contribution in https://github.com/ZuluSCSI/ZuluSCSI-firmware/pull/290

**Full Changelog**: https://github.com/ZuluSCSI/ZuluSCSI-firmware/compare/v2023.07.01...v2023.09.06

<hr /><em>This discussion was created from the release <a href='https://github.com/ZuluSCSI/ZuluSCSI-firmware/releases/tag/v2023.09.06'>v2023.09.06</a>.</em>

**Comment by Br0ther66** [comment]
Good morning, I wanted to dedicate these lines to the Zuluscsi community to express my joy, and also share this information that I think interesting, and that can help other users who have this same SCSI controller in Amiga 500.

Thanks to this update, an error that was chasing me since I bought the ZuluSCSI RP2040 has disappeared.

I bought the ZuluSCSI RP2040 in March 2023 for the SupraDrive 500XP controller of Amiga 500.
Everything related to mounting several units in the SD, including CD images works correctly. But until yesterday there was a problem related to the transfer of files from one device to another device in the SD Card. When copying files, the transference stops, and the operating system shows error 26.
The error could appear shortly after starting to copy, at half a minute, one minute, and so randomly over time.
I looked along all www about Error 26 in Amiga OS. I left my eyes without finding anything reference. There is information about all Amiga OS errors, but nothing for error 26 .... a mystery.

In June I updated the firmware dated 2023-06-04, but the error followed.
Yesterday I decided to update the firmware with the 2023-09-07 file and the error has disappeared completely using the same images, files, and configuration in the Zuluscsi.ini file. I have even copy an entire device to another device and it has never seemed again the mysterious error 26.

I do not know what has this firmware but solved the problem with the SupraDrive 500XP controller. I thought I would always have to live with it. The strange thing is that it also seems that I am the only one in the world to which this has happened.

I hope everything is understood even though my english is not very good.

Infinite thanks from Spain !!
Fran
